HyperVExtension/src/HyperVExtension/Templates/VmCredentialAdaptiveCardTemplate.json

```diff
@@ -0,0 +1,123 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.5",
+    "body": [
+        {
+            "type": "TextBlock",
+            "text": "${attemptCountText}",
+            "wrap": true
+        },
+        {
+            "type": "TextBlock",
+            "text": "${title}",
+            "horizontalAlignment": "Left",
+            "wrap": true,
+            "style": "heading"
+        },
+        {
+            "type": "TextBlock",
+            "text": "${description}",
+            "wrap": true
+        },
+        {
+            "type": "Input.Text",
+            "id": "UserVal",
+            "label": "${userNameLabel}",
+            "isRequired": true,
+            "errorMessage": "${userNameIsRequiredText}",
+            "$when": "${showOkButton == true}"
+        },
+        {
+            "type": "Input.Text",
+            "id": "PassVal",
+            "style": "Password",
+            "label": "${passwordLabel}",
+            "$when": "${showOkButton == true}"
+        },
+        {
+            "type": "Container",
+            "style": "attention",
+            "items": [
+                {
+                    "type": "Table",
+                    "columns": [
+                        {
+                            "width": "20px"
+                        },
+                        {
+                            "width": 1
+                        }
+                    ],
+                    "rows": [
+                        {
+                            "type": "TableRow",
+                            "cells": [
+                                {
+                                    "type": "TableCell",
+                                    "items": [
+                                        {
+                                            "type": "Image",
+                                            "url": "data:image/png;base64,${icon}",
+                                            "size": "Small"

```

Comment:
I believe Adaptive Cards lets you specify a pixel value here, and I suggest you do that. I've been meaning to do it for the other widgets, since these small/medium/large values differ between hosts. I know that's not a concern here, but it still protects you from any change in the hostconfig (where these values are specified).

Reply:
Yes, there is a pixel value here, but it doesn't do much in this case since I already set pixel value on the table column which gives the same result for the icon and keeps spacing between columns small (I also played with CollumnSet, but didn't get better result).